featurebyte/query_graph/node/metadata/operation.py

```diff
@@ -423,7 +434,7 @@ def table_ids(self) -> List[PydanticObjectId]:
         return list(set(table_ids))
 
 
-class OperationStructure(FeatureByteBaseModel):
+class OperationStructure(BaseModel):
```

Comment:
Will create another PR to update this to `dataclass` (https://featurebyte.atlassian.net/browse/DEV-1806).